bin/local.js

```diff
@@ -6,47 +6,47 @@
  *  Entry point into the client portion of sicksync
  */
 var fs = require('fs'),
-    watcher = require('chokidar'),
+    chokidar = require('chokidar'),
     path = require('path'),
     util = require('../lib/util'),
     WebSocketClient = require('../lib/ws-client'),
     bigSync = require('../lib/big-sync'),
     config = util.getConfig(),
     ignored = config.excludes,
-    isPaused = false,
-    devbox = null;
+    devbox = null,
+    watcher = null;
 
 require('colors');
 
-var NUM_FILES_FOR_LARGE_SEND = 10;
-var FILE_CHANGE_COOLDOWN_TIME = 10;
+var NUM_FILES_FOR_LARGE_SEND = 3;
+var FILE_CHANGE_COOLDOWN_TIME = 20;
 
 var rebouncedFileChange = util.rebounce(onFileChange, onBigTransfer, NUM_FILES_FOR_LARGE_SEND, FILE_CHANGE_COOLDOWN_TIME);
 
 function onBigTransfer() {
     if (config.debug) console.log('[local] Sending large change');
-
+    
+    stopFileWatch();
     bigSync(onBigTransferDone);
-    isPaused = true;
 }
 
 function onBigTransferDone() {
     if (config.debug) {
         console.log('['+ config.hostname +'] Received large change');
+        startFileWatch();
```

Comment:
Looks to me like you only startFileWatch again if config.debug is true? Don't we want that in either case?